sklearn/linear_model/tests/test_sgd.py

```diff
@@ -823,7 +824,27 @@ def test_multiple_fit(self):
         y = [["ham", "spam"][i] for i in LabelEncoder().fit_transform(Y)]
         clf.fit(X[:, :-1], y)
 
-
+    def test_loss_param_grid_search(self):
+        # Make sure the predict_proba works when loss is specified
+        # as one of the parameters in the param_grid
+        param_grid = {
+            'loss': ['log'],
+        }
+        clf_grid = GridSearchCV(estimator=self.factory(random_state=42),
+                                param_grid=param_grid)
+        clf_grid.fit(X, Y)
+        clf_grid.predict_proba(X)
+        clf_grid.predict_log_proba(X)
+        
+    def test_not_fitted_error(self):
+        # Make sure raise not fitted error and not the loss='hinge' error
+        clf = self.factory()
+        assert_raises_regexp(_NotFittedError,
+                             "This {} instance is not fitted yet. "
```

Comment:
python 2.6 requires numbered fields with `format`, hence why travis is a bit angry